extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -188,13 +195,21 @@ export class SubscriptionService {
    * @return {!Promise<!./entitlement.Entitlement>}
    */
   fetchEntitlements_(subscriptionPlatform) {
-    return subscriptionPlatform.getEntitlements().then(entitlement => {
-      if (!entitlement) {
-        entitlement = Entitlement.empty(subscriptionPlatform.getServiceId());
-      }
+    return this.timer_.timeoutPromise(
+        SERVICE_TIMEOUT,
```

Comment:
Can you please right away do something like this:
`SERVICE_TIMEOUT * (getMode().development ? 2 : 1)`?

Reply:
Done